### PR TITLE
Refactor checkboxes, settings. tricks and fonts

### PR DESCRIPTION
**Pull request by Hamelatoire**
Change 1: Upgraded the fonts and responsiveness a bit 
image
Change 2: Refactored the checkbox component and added styles
image

Change 3: Replaced the flex in the settings for a grid system
image

Chang 4: Split the tricks into two lists
image

### CODE REVIEW

File: app/components/App.jsx [hunk 1]
```diff
@@ -5,9 +5,9 @@ import { Progress } from './Progress';
 import { Result } from './Result';
 
 export const App = () => {
-  const [error, setError] = useState("");
```
Comment (Hamelatoire) [inline app/components/App.jsx:8]: this file is just linting and removing a ccs class that i removed

File: app/components/Progress.jsx [hunk 2]
```diff
@@ -1,7 +1,5 @@
 import React from 'react';
```
Comment (Hamelatoire) [inline app/components/Progress.jsx:2]: More linting

File: app/components/Settings.jsx [hunk 3]
```diff
@@ -11,27 +11,31 @@ export const Settings = ({ category, settings, setSetting }) => {
...
-          <Dropdown
-            value={settings[setting.key]}
-            key={setting.key}
-            label={setting.name}
-            options={setting.values}
-            onChange={(v) => setSetting({ [setting.key]: v })}
-          />
-        ))}
-      </div>
+      {booleanList.length > 0 && (
```
Comment (Hamelatoire) [inline app/components/Settings.jsx:14]: Added conditional rendering cause the div was adding margins for nothing when it was empty

File: app/style/_global.css [hunk 4]
```diff
@@ -7,10 +7,22 @@ html, body {
 }
 
 html {
-  font-family: Helvetica, Arial, sans-serif;
-  font-size: 62.5%;
+  font-family: Sailec, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+  font-size: 62%;
+  -webkit-font-smoothing: antialiased;
```
Comment (Hamelatoire) [inline app/style/_global.css:12]: those last two are probably not doing much

File: app/style/generator.css [hunk 5]
```diff
@@ -1,8 +1,3 @@
-.generator-title {
```
Comment (Hamelatoire) [inline app/style/generator.css:1]: now using the generic h1 style

**Comment by Hamelatoire** [comment]
@Nax just fixed the font size thing

**Comment by Nax** [comment]
I don't agree with every change - I'll still merge it because it goes in the right direction tho :)

**Pull request merged**